### PR TITLE
API Aspects Endpoint Feature Complete With Full Tests 

### PR DESCRIPTION
**Pull request by HankG**

### CODE REVIEW

File: spec/integration/api/aspects_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+        expect(response.body).to eq(I18n.t("api.endpoint_errors.aspects.cant_update"))
+      end
+
+      it "fails with bad id" do
+        patch(
+          api_v1_aspect_path("-1"),
+          params: {name: "NewAspectName", access_token: access_token}
+        )
+
+        expect(response.status).to eq(422)
```
Comment (frankrousseau) [inline spec/integration/api/aspects_controller_spec.rb]: Maybe that 404 would be more accurate.

**Review by frankrousseau — COMMENTED** [review]
Ready to merge. I have only one question about 422 errors vs 404.

File: spec/integration/api/aspects_controller_spec.rb (hunk 1 above)
Comment (HankG) [reply spec/integration/api/aspects_controller_spec.rb]: I believe you are correct.  Let me look into that.

Comment (HankG) [reply spec/integration/api/aspects_controller_spec.rb]: Fixed in current version

**Comment by HankG** [comment]
Squashed and ready to go :)

**Pull request merged**